include/swift/Driver/ParseableOutput.h

```diff
@@ -29,15 +29,19 @@ class Job;
 namespace parseable_output {
 
 /// \brief Emits a "began" message to the given stream.
-void emitBeganMessage(raw_ostream &os, const Job &Cmd, int64_t Pid);
+void emitBeganMessage(raw_ostream &os, const Job &Cmd, int64_t Pid,
+                      sys::TaskProcessInformation Process);
```

Comment:
Echoing Graydon, `Process` is the wrong name here. I'd be fine with `TPI` (`pi` could be interpreted as 3.14) or `processInfo`.
Also it would be nice to be consistent (locally) on matters of casing. In other words, either `OS` instead of `os`, or `cmd` and `pid` and `tpi`. (Sigh, the casing is a nitpick, I know.)

Reply:
I renamed it to ProcessInfo already, just want to fix the other concerns as well before pushing it :) Interpreting `pi` as the number `pi` is an interesting thought though :D It could also be interpreted as 🍰.
I didn't create the signature of the function but it seems to be convention in llvm to write variables in UpperCamelCase, `os` however refers better to output stream than `OS` which could indicate operating system. It's not ideal but I would like to keep it like that.

Reply:
I think I've seen `OS` for output stream, and I keep thinking operating system, yes. Feel free to keep it as is.